freetextresponse/tests.py

```diff
@@ -56,7 +57,12 @@ def make_an_xblock(cls, **kw):
         Helper method that creates a Free-text Response XBlock
         """
         course_id = SlashSeparatedCourseKey('foo', 'bar', 'baz')
-        runtime = Mock(course_id=course_id)
+        runtime = Mock(
+            course_id=course_id,
+            service=Mock(
+                return_value=Mock(_catalog={}),
```

Comment:
What is this used by?

Reply:
So the `i18n` service doesn't freak out. I've added an inline comment to clarify.

Reply:
Thanks @OmarIthawi